src/pages/Home/Home.page.jsx

```diff
@@ -1,12 +1,18 @@
 import CardList from 'components/CardList/CardList.component';
-import React from 'react';
+import { useYoutubeVideos } from 'hooks/youtube/videos/useYoutubeVideos';
+import React, { useEffect } from 'react';
 import { HomeContainer } from './HomeElements.styled';
 
 function HomePage() {
+  const { videos, isLoading, fetchHomeVideos } = useYoutubeVideos();
+  useEffect(() => {
+    fetchHomeVideos();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
   return (
     <HomeContainer>
       <h1>Welcome to the Challenge!</h1>
-      <CardList />
+      {isLoading || !videos.length ? <h3>Loading...</h3> : <CardList videos={videos} />}
```

Comment:
I really liked that you added a loading fragment, this makes it look more professional